### PR TITLE
Seismic vis (Obspy Integration)

### PR DESCRIPTION
**Pull request by eusoubrasileiro**
Reopening  [PR 61](https://github.com/leouieda/fatiando/pull/61) and  [PR 126](https://github.com/leouieda/fatiando/pull/61) *Obspy integration.*

Simpler cookbook example where Obspy is just imported.

screenshot from 2015-04-15 15 23 26

- [x] Make tests for new code  (*ok then*)
- [x] Create/update doc-strings 
- [x] Update requirements 
- [x] Code follows PEP8 style conventions 
- [x] Code and docs have been spell-checked (*perfect*)
- [x] Changelog entry (doing now)
- [x] Documentation builds properly (*builds properly*)
- [x] All tests pass
- [x] Can be merged

### CODE REVIEW

**Comment by leouieda** [comment]
:+1:

**Comment by eusoubrasileiro** [comment]
Rsrs I got it hahah

File: cookbook/vis_mpl_seismic.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""
+Seismic: seismic plotting utilities uses Obspy for SEGY reading
```
Comment (leouieda) [inline cookbook/vis_mpl_seismic.py]: @eusoubrasileiro could you change the title to something like "plotting a seismic section"? The title should be something that a person will search for. Just to be clear, most of my titles for recipes are awful and need to be changed.

File: cookbook/vis_mpl_seismic.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from fatiando.vis import mpl
+from obspy.segy import segy
+import urllib
+import numpy as np
+
+# fetch sample SEGY data, near-offset marmousi data
+url = "http://dl.dropboxusercontent.com/" \
+      "s/i287ci4ww3w7gdt/marmousi_nearoffset.segy"
+urllib.urlretrieve(url, 'marmousi_nearoffset.segy')
+segyfile = segy.readSEGY('marmousi_nearoffset.segy')
```
Comment (leouieda) [inline cookbook/vis_mpl_seismic.py]: Could you add a comment above this line saying: "We'll use the obspy library to load the SEGY data"

File: fatiando/vis/mpl.py [hunk 3]
```diff
@@ -992,3 +994,93 @@ def pcolor(x, y, v, shape, interp=False, extrapolate=False, cmap=pyplot.cm.jet,
         plot = basemap.pcolor(lon, lat, V, cmap=cmap, vmin=vmin, vmax=vmax,
                               picker=True)
     return plot
+
+
+def seismic_wiggle(section, dt=0.004, ranges=None, scale=1.,
+                   color='k', normalize=False):
+    """
+    Plot a seismic section (numpy 2D array matrix) as wiggles.
+    Slow for more than 200 traces, in this case use `seismic_image`.
```
Comment (leouieda) [inline fatiando/vis/mpl.py]: Add a  line between these two sentences and make the second line a warning with:

    Plot a seismic section (numpy 2D array matrix) as wiggles.

    .. warning::        
        Slow for more than 200 traces, in this case use ``seismic_image``.
   
Notice that the format for code in rst is `` (not ` in Markdown). I do this all the time.

**Comment by leouieda** [comment]
@eusoubrasileiro don't worry about the tests for now. We need to get all of `fatiando.vis`  tested  eventually but this will need a larger effort.

**Comment by leouieda** [comment]
@eusoubrasileiro sorry I didn't the comment about tesseroids. The tesseroids tests are a bit slow but not as much as you're experiencing. The new code uses numba for speedups. If it's not installed, it  runs the slow numpy version instead. So install numba with  `conda install numba` and you'll see a very large speedup in the tests.

**Comment by leouieda** [comment]
You can tick off the requirements item as well.

File: cookbook/vis_mpl_seismic.py (hunk 1 above)
Comment (eusoubrasileiro) [inline cookbook/vis_mpl_seismic.py]: ok got it.

**Comment by eusoubrasileiro** [comment]
It`s done.

**Comment by eusoubrasileiro** [comment]
although I've installed numba when I run `make test` I am getting the following error:

```
======================================================================
ERROR: gravmag.tesseroid numba and pure python implementations give same result
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/andre/PycharmProjects/fatiando/test/test_gravmag_tesseroid.py", line 89, in test_numba_vs_python
    nb = func(lon, lat, height, model, engine='numba')
  File "/home/andre/PycharmProjects/fatiando/fatiando/gravmag/tesseroid.py", line 383, in potential
    ratio=ratio, engine=engine, njobs=njobs, pool=pool)
  File "/home/andre/PycharmProjects/fatiando/fatiando/gravmag/tesseroid.py", line 277, in _dispatcher
    field])
  File "/home/andre/PycharmProjects/fatiando/fatiando/gravmag/tesseroid.py", line 301, in _forward_model
    module = _get_engine(engine)
  File "/home/andre/PycharmProjects/fatiando/fatiando/gravmag/tesseroid.py", line 224, in _get_engine
    module = _tesseroid_numba
NameError: global name '_tesseroid_numba' is not defined

----------------------------------------------------------------------
Ran 174 tests in 3504.121s

FAILED (errors=1)
make: *** [test] Error 1
```

am I missing something...?

File: cookbook/vis_mpl_seismic.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+"""
+Seismic: plotting a seismic section from a SEGY
+
+Uses ObsPy package for SEGY file reading
```
Comment (leouieda) [inline cookbook/vis_mpl_seismic.py]: @eusoubrasileiro final thing, I promise! Could you also add a line here saying something like "you can find instructions for installing Obspy at http://obspy.org/"?

Comment (eusoubrasileiro) [inline cookbook/vis_mpl_seismic.py]: Sure, done

File: cookbook/vis_mpl_seismic.py (hunk 2 above)
Comment (eusoubrasileiro) [inline cookbook/vis_mpl_seismic.py]: sure, done

**Comment by leouieda** [comment]
@eusoubrasileiro perfect!

**Pull request merged**